pkg/appendable/index_file_test.go

```diff
@@ -61,6 +50,47 @@ func TestIndexFile(t *testing.T) {
 
 	})
 
+	t.Run("compare mock index file after appending", func(t *testing.T) {
```

Comment:
we now check for index file equality after appending an extra data line to both formats